sep-template.md

```diff
@@ -40,3 +40,14 @@ objections or concerns raised during discussion.
 ## Security Concerns
 All SEPs should carefully consider areas where security may be a concern, and document them
 accordingly. If a change does not have security implications, write "N/A".
+
+## Changelog
+All SEPs should contain a changelog that succinctly lists its versions and what change was introduced in each version. Information included should contain a succinct by complete description of what has changed. Succinct is important so that over time a large changelog is easily digested.
+
+Example:
+
+- `v0.1.0`: Initial draft.
```

Comment:
How about also linking to the commit or PR that incremented the document to the associated version? This would allow each SEP to effectively have a git commit history embedded in the file, which I think is valuable because looking at the stellar-protocol commit history has commits to many other documents.

Reply:
Added to the example in 4210cb279c11a95aee6a9bdf7025d7806e85d34a.